File: string/reverseInteger.py
# https://leetcode.com/problems/reverse-integer/
class Solution:
    def reverse(self, x: int) -> int:
        negFlag = 1
        if x < 0:
            negFlag = -1
            strx = str(x)[1:]
        else:
            strx = str(x)

        x = int(strx[::-1])

        if x > pow(2, 31):
            return 0
        else:
            return x * negFlag

File: string/reverseInteger.js
// https://leetcode.com/problems/reverse-integer/
/**
 * @param {number} x
 * @return {number}
 */
const reverse = (x) => {
  const reversedNumber =
    parseInt(x.toString().split("").reverse().join("")) * Math.sign(x);
  if (Math.abs(reversedNumber) > 2147483648) {
    return 0;
  }
  return reversedNumber;
};
